package.json

```diff
@@ -58,7 +59,9 @@
     "grunt-svgstore": "0.5.0",
     "jsass-vars": "0.0.3",
     "load-grunt-config": "0.16.0",
-    "perfjankie": "1.2.2"
+    "perfjankie": "1.2.2",
+    "promise": "^7.0.3",
+    "selenium-webdriver": "^2.46.1"
```

Comment:
Not sure what to do about this. We need code to inject axe-core into the pages being tested and that code exists in axe-webdriverjs but that requires a WebDriver instance which comes from selenium-webdriver.

The way that selenium is being included right now, we cannot get a WebDriver instance from it, so we have a set of options:

1) Include a dependency the way I did, or
2) Write code similar to the code in axe-webdriverjs to inject axe-core into the test pages

Pro for 1 is less code to maintain for this project, con is more disk space used by duplicate selenium JARs.

Given that we are using Node, we will have duplicate packages because that is one of the side-effects of the NPM package system, so I think this is not a big deal